packages/react-app/.env.example

```diff
@@ -1,8 +1,14 @@
-REACT_APP_INFURA_ID=YOUR_INFURA_ID
-REACT_APP_API_KEY=
-REACT_APP_AUTH_DOMAIN=
-REACT_APP_PROJECT_ID=
-REACT_APP_STORAGE_BUCKET=
-REACT_APP_MESSAGING_SENDER_ID=
-REACT_APP_APP_ID=
-REACT_APP_MEASUREMENT_ID=
\ No newline at end of file
+REACT_APP_INFURA_ID=# your infura id
+REACT_APP_API_KEY=# your firebase api key
+REACT_APP_AUTH_DOMAIN=# your firebase auth domain
+REACT_APP_PROJECT_ID=# your firebase project id
+REACT_APP_STORAGE_BUCKET=# your firebase storage bucket
+REACT_APP_MESSAGING_SENDER_ID=# your firebase messaging sender id
+REACT_APP_APP_ID=# your firebase app id
+REACT_APP_MEASUREMENT_ID=# your firebase measurement id
+REACT_APP_CLIENT_ID=# your unstoppable domain client id
+REACT_APP_CLIENT_SECRET=# your unstoppable domain client secret
+REACT_APP_REDIRECT_URIS=# your unstoppable domain redirect uris
```

Comment:
```
REACT_APP_CLIENT_ID=# your unstoppable domain client id
REACT_APP_CLIENT_SECRET=# your unstoppable domain client secret
REACT_APP_REDIRECT_URIS=# your unstoppable domain redirect uris
```

@chunza2542 if you need this for testing please DM me on discord spiritbro1#1101 or email me 
on 3aml5sclu@mozmail.com will give you that value for environment variable

Reply:
I sent the application request for this already, maybe it'll take a few days.

Reply:
@chunza2542 no need for waiting it can be used straight away even if it said pending

Reply:
@spiritbro1 I've deployed it. Could you try it at https://ethtalk.app?

Reply:
@chunza2542 thank you for deploying, however i got this error : 

![image](https://user-images.githubusercontent.com/62529025/153214693-bb1c875a-acfa-4520-a1a4-b94f0825105d.png)

i think the error could be the `New Redirect URIs` in unstoppable domain application could you please check that the `New Redirect URIs` is `https://ethtalk.app/callback` ? like this:

![image](https://user-images.githubusercontent.com/62529025/153215919-10d1c933-ceda-4b36-9371-763e7a19e875.png)

and make sure your `Url` is the same as your `New Redirect URIs` because when you click save the `New Redirect URIs` url will always change to url of `Url` this two should be the same : 

![image](https://user-images.githubusercontent.com/62529025/153216491-475ca859-b62c-44be-9a5e-5252f800b042.png)

and this is how you put it into `.env` : 

```
REACT_APP_REDIRECT_URIS=https://ethtalk.app/callback
```

Reply:
@spiritbro1 Updated, could you try it again?

Reply:
@chunza2542 now the error is this : 

![image](https://user-images.githubusercontent.com/62529025/153373670-cf74ce7d-6688-4314-b7b6-cc11fadb5ff9.png)

Reply:
@spiritbro1 sry for the late reply, but It seems like I forgot to deploy the cloud functions 😓.